### DISCUSSION TITLE
CAPTCHA wording needs to change

### DISCUSSION THREAD

**Discussion by madcodemonkey** [community/community#83984] [Other Feature Feedback, Questions, & Ideas]
While configuring "SMS/Text message" 2 factor authentication, I had to do a CAPTCHA  that involved a finger and devices.  It states:
"Use the arrows to rotate the object to face in the direction of the hand."  
If you literally follow that instruction, you should turn the image to point to left always because the hand is the picture on the left side.  

I suggest changing the wording of the message to 
"Use the arrows to rotate the object to face the SAME DIRECTION that the finger is POINTING".  

With this new wording the number of failures should go down.

**Comment by Lana777** [comment]
That NEVER works! I tried multiple times with both scenarios. That is bugged.. , broken.
